### DISCUSSION TITLE
Does running your own PDS mean you can host whatever legal?

### DISCUSSION THREAD

**Discussion by kemopup** [bluesky-social/pds#148] [Q&A] [answered]
I love Bluesky, but when it comes to what artwork is allowed, things get a bit dicey. Furries are okay BUT must appear fully adult and are four legged furries allowed? Maybe... Might promote bestiality??? 

So I want to avoid all that and host a PDS that's a safe space for illustrators to post art they want without worrying about being banned. Can Bluesky kill my PDS if they don't like the content on it? 

tl;dr do the same ToS apply to PDS?

**Comment by hooverdc** [comment, marked as answer]
Their AppView can refuse to aggregate your content, I believe.
